.github/workflows/buildpack-ubuntu2004.yml

```diff
@@ -0,0 +1,46 @@
+name: buildpack:ubuntu2004
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/buildpack-ubuntu2004.yml'
+      - 'scripts/docker/buildpack/Dockerfile.ubuntu2004'
+jobs:
+  build:
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_DOCKER_REPOSITORY: docker.pkg.github.com/${{ github.repository }}
+      IMAGE_NAME: buildpack
+      IMAGE_VARIANT: ubuntu2004
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check version
+        run: .github/workflows/buildpack/check-version.sh
+
+      - name: Build image
+        run: docker build scripts/docker/${IMAGE_NAME} --file scripts/docker/${IMAGE_NAME}/Dockerfile.$IMAGE_VARIANT --tag $IMAGE_NAME
+
+      - name: Test image
+        run: .github/workflows/buildpack/test-image.sh
+
+      - name: Push image
+        run: .github/workflows/buildpack/push-image.sh
+
+      - name: Prepare action
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: Build action
+        run: npm i && npm run build
+        working-directory: ./.github/actions/pr-comment
```

Comment:
If we'd use a ready-made action for this, we wouldn't need this, would we?

Reply:
i will check